src/Microsoft.Framework.Caching.Abstractions/IMemoryCache.cs

```diff
@@ -16,24 +16,24 @@ public interface IMemoryCache : IDisposable
         /// <summary>
         /// Create or overwrite an entry in the cache.
         /// </summary>
-        /// <param name="key">A string identifying the entry. This is case sensitive.</param>
+        /// <param name="key">An object identifying the entry.</param>
```

Comment:
Should we remind here that is a good idea to use structurally comparable objects for keys? I guess it depends on the target audience.